analyses/create-corplots/01-process-data.Rmd

```diff
@@ -0,0 +1,270 @@
+# Storing all Deceased, Recurrence Sample pairs in a list so each file can be processed iteratively
+scratch_Rec_Dec <-
+  file.path(scratch, "Rec_Dec")
+if (!dir.exists(scratch_Rec_Dec)) {
+  dir.create(scratch_Rec_Dec)
+}
+
+setwd(scratch_Rec_Dec)
+data <- maf_files_Deceased %>% 
+  filter(descriptors == "Deceased, Recurrence")
+
+# we will split maf data by the "Kids_First_Participant_ID" column
+data %>% 
+  group_split(Kids_First_Participant_ID) %>% 
+  walk(~write_csv(.x, paste0(.x$Kids_First_Participant_ID[1], "-vaf.csv")))
+
+##########################################################################################  
+# Storing all Dx_Pro_Dec Sample pairs in a list so each file can be processed iteratively
+scratch_Dx_Pro_Dec <-
+  file.path(scratch, "Dx_Pro_Dec")
+if (!dir.exists(scratch_Dx_Pro_Dec)) {
+  dir.create(scratch_Dx_Pro_Dec)
+}
+
+setwd(scratch_Dx_Pro_Dec)
+data <- maf_files_Deceased %>% 
+        mutate(data = case_when(grepl("Deceased, Diagnosis, Diagnosis, Progressive, Progressive", descriptors) ~ "Dx_Pro_Dec")) %>%
+ filter(data == "Dx_Pro_Dec")
+
+# we will split maf data by the "Kids_First_Participant_ID" column
+data %>% 
+  group_split(Kids_First_Participant_ID) %>% 
+  walk(~write_csv(.x, paste0(.x$Kids_First_Participant_ID[1], "-vaf.csv")))
+
+
+##########################################################################################
+# Storing all Dx_Rec_Dec Sample pairs in a list so each file can be processed iteratively
+scratch_Dx_Rec_Dec <-
+  file.path(scratch, "Dx_Rec_Dec")
+if (!dir.exists(scratch_Dx_Rec_Dec)) {
+  dir.create(scratch_Dx_Rec_Dec)
+}
+
+setwd(scratch_Dx_Rec_Dec)
+data <- maf_files_Deceased %>% 
+  mutate(data = case_when(grepl("Deceased, Diagnosis, Recurrence", descriptors) ~ "Dx_Rec_Dec")) %>%
+  filter(data == "Dx_Rec_Dec")
+
+# we will split maf data by the "Kids_First_Participant_ID" column
+data %>% 
+  group_split(Kids_First_Participant_ID) %>% 
+  walk(~write_csv(.x, paste0(.x$Kids_First_Participant_ID[1], "-vaf.csv")))
+
+##########################################################################################
+# Storing all Pro_Rec_Dec Sample pairs in a list so each file can be processed iteratively
+scratch_Pro_Rec_Dec <-
+  file.path(scratch, "Pro_Rec_Dec")
+if (!dir.exists(scratch_Pro_Rec_Dec)) {
+  dir.create(scratch_Pro_Rec_Dec)
+}
+
+setwd(scratch_Pro_Rec_Dec)
+data <- maf_files_Deceased %>% 
+ mutate(data = case_when(grepl("Deceased, Progressive, Recurrence", descriptors) ~ "Pro_Rec_Dec")) %>%
+  filter(data == "Pro_Rec_Dec")
+
+# we will split maf data by the "Kids_First_Participant_ID" column
+data %>% 
+  group_split(Kids_First_Participant_ID) %>% 
+ walk(~write_csv(.x, paste0(.x$Kids_First_Participant_ID[1], "-vaf.csv")))
+##########################################################################################
+# let's change back to the root_dir for the rest of the module
+setwd(root_dir)
```

Comment:
```suggestion
```

really never a reason to use setwd within the rmd files when you find the root dir, we use the file.path() command to determine where everything should be pulled in from and saved to. You will have to adjust your input/output commands accordingly